reloj_checador_team03/app/views/branches/_form.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="modal-dialog">
 <% @branch= Branch.new if @branch.nil?  %>
```

Comment:
Not part of this change, but try to have as little as possible this kind of logic from the views
You can instantiate this from the `Branches#new` definition 😄